airbyte-integrations/connectors/source-monday/source_monday/manifest.yaml

```diff
@@ -210,3 +295,9 @@ streams:
 check:
   stream_names:
     - "users"
+
+# The smallest limit for 1 minute is 250 queries, so set the default number of workers to 4. https://developer.monday.com/api-reference/docs/rate-limits#minute-limit
+concurrency_level:
+  type: ConcurrencyLevel
+  default_concurrency: "{{ config.get('num_workers', 4) }}"
+  max_concurrency: 40
```

Comment:
nit, did you come across why 40 is the max workers?

Reply:
Let's replace this number. Monday actually has a pretty detailed section on rate limits: https://developer.monday.com/api-reference/docs/rate-limits

`40` actually seems like an acceptable threshold since they call out only allowing for at most 40 concurrent requests at any given moment. But let's directly reference https://developer.monday.com/api-reference/docs/rate-limits#concurrency-limit this in the comment